src/pages/sign-up-success-page/index.js

```diff
@@ -1 +1,13 @@
 import './index.scss';
```

Comment:
if there is no email or is visited flag is true in local storage redirect user to home page according to selected language

Reply:
fixed in [implement better logic for redirects](https://github.com/shpp/programming.org.ua/pull/106/commits/3f63c446a60b2a28207194f0e543909c3348af67)